.github/workflows/test.yml

```
@@ -19,29 +19,47 @@ jobs:
   # All tests under this job are run with pre-computed libR-sys bindings.
   tests_no_bindgen:
     runs-on: ${{ matrix.config.os }}
-
     name: ${{ matrix.config.os }} (R-${{ matrix.config.r }} rust-${{ matrix.config.rust-version }})
-
     strategy:
       fail-fast: false
       matrix:
         config:
-          - { os: windows-latest, r: 'release', rust-version: 'stable-msvc', rtools-version: '44' }
-          - { os: windows-latest, r: 'devel',   rust-version: 'stable-msvc', rtools-version: '44' }
-          - { os: windows-latest, r: 'oldrel',  rust-version: 'stable-msvc', rtools-version: '43' }
-
-          - { os: macOS-latest,   r: 'release', rust-version: 'stable' }
-          # - {os: macOS-latest,   r: 'release', rust-version: 'nightly'}
-          # - {os: macOS-latest,   r: 'devel', rust-version: 'stable'}
-          # - {os: macOS-latest,   r: 'oldrel', rust-version: 'stable'}
+          - {
+              os: windows-latest,
+              r: "release",
+              rust-version: "stable-msvc",
+              rtools-version: "44",
+            }
+          - {
+              os: windows-latest,
+              r: "devel",
+              rust-version: "stable-msvc",
+              rtools-version: "44",
+            }
+          - {
+              os: windows-latest,
+              r: "oldrel",
+              rust-version: "stable-msvc",
+              rtools-version: "43",
+            }
 
-          - { os: ubuntu-latest,   r: 'release', rust-version: 'stable', check_fmt: true }
-          - { os: ubuntu-latest,   r: 'release', rust-version: 'nightly' }
+          - { os: macOS-latest, r: "release", rust-version: "stable" }
+          - {
+              os: ubuntu-latest,
+              r: "release",
+              rust-version: "stable",
+              check_fmt: true,
+            }
+          - {
+              os: ubuntu-latest,
+              r: "release",
+              rust-version: "nightly-msvc",
```

on ubuntu, the rust-version should be `"nightly"`
```suggestion
              rust-version: "nightly",
```